core/types/arb_types.go

```diff
@@ -3,17 +3,32 @@ package types
 import (
 	"context"
 	"encoding/binary"
-	"errors"
 	"fmt"
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/ethereum/go-ethereum/log"
 
 	"github.com/ethereum/go-ethereum/common"
 )
 
-var ErrUseFallback = errors.New("missing trie node 0000000000000000000000000000000000000000000000000000000000000000 (path )")
+type fallbackError struct {
+}
+
+var fallbackErrorMsg = "I seem to be missing trie node 0000000000000000000000000000000000000000000000000000000000000000 (path )"
+var fallbackErrorCode = -32001
```

Comment:
I think the default should be the same as geth to maximize compatibility, with the option of customizing it